content/_partners/faq._en.md

```diff
@@ -64,7 +64,7 @@ general_accordion:
         id: a-6
         title: How do we partner with Login.gov?
         content: >-
-            <a href="https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e" class="external-link">To get started, reach out to our team using this form</a>. We’ll work with you to understand and capture your needs and requirements at a high level. Together, we’ll decide whether Login.gov makes sense for your particular business and use cases. If we decide to move forward, the next step is to sign an Interagency Agreement (IAA). This signals a mutual commitment which allows us to commit further resources to technical discovery and integration and migration planning.
+            <a href="https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e" class="external-link">Contact an account manager to get started.</a> We’ll work with you to understand and capture your needs and requirements at a high level. Together, we’ll decide whether Login.gov makes sense for your particular business and use cases. If we decide to move forward, the next step is to sign an Interagency Agreement (IAA). This signals a mutual commitment which allows us to commit further resources to technical discovery and integration and migration planning.
```

Comment:
Q: is our style to put a period inside of a link? (My personal style would be to leave out but unsure if we have notes as a team on this)

```suggestion
            <a href="https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e" class="external-link">Contact an account manager to get started</a>. We’ll work with you to understand and capture your needs and requirements at a high level. Together, we’ll decide whether Login.gov makes sense for your particular business and use cases. If we decide to move forward, the next step is to sign an Interagency Agreement (IAA). This signals a mutual commitment which allows us to commit further resources to technical discovery and integration and migration planning.
```

Reply:
Thanks for catching that. I don't think we have a prescribed style, but looking at other static site links, it looks like the period isn't within the link. So it makes to put it outside.